.gitignore

```diff
@@ -0,0 +1,132 @@
+# Logs
```

Comment:
It would be a good idea to check how many of the files listed in here are actually present in your project. If they are not there (such as the coverage folder, `logs`, `*.logs` etc), they can be removed from this file